tests/e2e/.env

```diff
@@ -11,22 +11,22 @@ E2E_RUN_KVTOOL_NETWORKS=true
 E2E_KVTOOL_KAVA_CONFIG_TEMPLATE="master"
 
 # E2E_INCLUDE_IBC_TESTS when true will start a 2nd chain & open an IBC channel. It will enable all IBC tests.
-E2E_INCLUDE_IBC_TESTS=true
+E2E_INCLUDE_IBC_TESTS=false
```

Comment:
why no ibc tests?

Reply:
Re-enabled 😃 Might have disabled to make the upgrade height lower to be faster